components/network/pdns-recursor/Makefile

```diff
@@ -0,0 +1,59 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Josef 'Jeff' Sipek <jeffpc@josefsipek.net>
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		pdns-recursor
+COMPONENT_VERSION=	3.7.3
+COMPONENT_PROJECT_URL=	https://www.powerdns.com/downloads.html
+COMPONENT_FMRI=		service/network/dns/powerdns-recursor
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.bz2
+COMPONENT_ARCHIVE_HASH=	\
+	sha256:859ca6071147dd2e2ac1b2a5c3d5c2cbff0f5cbc501660db4259e7cbf27fea11
+COMPONENT_ARCHIVE_URL=	http://downloads.powerdns.com/releases/$(COMPONENT_ARCHIVE)
+COMPONENT_LICENSE=	GPLv2
+COMPONENT_LICENSE_FILE=	$(COMPONENT_NAME).license
+COMPONENT_SUMMARY=	a DNS server
+COMPONENT_CLASSIFICATION="System/Services"
+
+include ../../../make-rules/prep.mk
+include ../../../make-rules/configure.mk
+include ../../../make-rules/ips.mk
+
+CXXFLAGS=-g
```

Comment:
To make debugging possible. :)  Since powerdns tends to uncover illumos shortcomings, I found having dwarf debug info available quite useful.

On a related note, once Joyent's CTF changes fully land in the gate, mdb & friends will be able to use dwarf debug info instead of requiring build-time CTF conversion.  Yes, for some applications, the dwarf data will be prohibitively large.